### ISSUE TITLE
Enable speaking in special views

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1355]
This would be cool - listen to a verse(s) in all the translations.

**Comment by tuomas2** [comment]
Speaking of special documents is a special task.. :-)

**Comment by tuomas2** [comment]
Let's see when I'll be motivated to do that.

**Comment by tuomas2** [comment]
Similarly it would be cool to have speaking implemented for study pads, my notes view, multidocument view (same as comparison),...

**Comment by agrogers** [comment]
Yes, definitely! I'm probably less interested in my own personal notes but
would be great to have all the verses read out.

Might need to think about how the references are said. I don't really want
to hear the words 'chapter' and 'verse' 100 times. 'Matthew five
forty-eight' would be better than 'Matthew chapter five verse forty-eight'

**Comment by agrogers** [comment]
No church camps for you until you get that done 😏

**Comment by tuomas2** [comment]
> Yes, definitely! I'm probably less interested in my own personal notes but would be great to have all the verses read out. Might need to think about how the references are said. I don't really want to hear the words 'chapter' and 'verse' 100 times. 'Matthew five forty-eight' would be better than 'Matthew chapter five verse forty-eight'

Study pads and personal notes might become shared documents / another "commentary" format (due to export/import) in the long run, so they might not be just your personal notes that you read :-)

**Comment by tuomas2** [comment]
> No church camps for you until you get that done 

Sorry, this is now sunk into the deep waters of the wishing well :-)

**Comment by agrogers** [comment]
Haha. I like the idea. Indeed that night be another way to enter notes. You
provide a 'Notes' commentary window that shows the notes attached to the
verse you are looking at. And if there are no notes it shows a plus so you
can add one. Is that what you are saying?

I kind of like the idea of the notes being stored online in a public
repository that doesn't need to be downloaded. And people can vote them up
so the best ones rise to the top. But I think you have a very offline
approach don't you? And I don't know who would use them. How many
commentaries do we need?

**Comment by tuomas2** [comment]
I think I want to keep AB offline app, but it does not prevent sharing docs between users or even making "user documents" repository where users can suggest their docs to be published. 

Voting is a bit dangerous idea when it comes to theological docs. I don't want heresies to prosper just because they become popular :-P

**Comment by tuomas2** [comment]
https://support.andbible.org/scp/tickets.php?id=1653
